gaia-fast-list.js

```diff
@@ -173,22 +177,18 @@ var GaiaFastListProto = {
         color: var(--title-color);
       }
 
+      ::content .gfl-section {
+        position: relative;
+        padding-top: 20px;
```

Comment:
It appears from the [visual spec](https://mozilla.app.box.com/s/t4417btikyek440uxtkb3wfoue8umk9c) (page 18) that the first section has the same margin above.